docs/walkthrough.md

```diff
@@ -310,19 +312,22 @@ Take the resulting value, and place it into the following file:
 
 <summary>cm-registration.yaml</summary>
 
+*Note that apiregistration moved to stable in 1.10, so you'll need to use
+the `apiregistration.k8s.io/v1` API version there*.
```

Comment:
I was not aware of this, neat!

Reply:
https://github.com/kubernetes/kubernetes/pull/58393